### ISSUE TITLE
[Suggestion] Right-click file -> create symlink (context menu)

### ISSUE THREAD

**Issue by onlyreportingissues** [pop-os/cosmic-files#566]
An option to create a symlink of a file using the right-click context menu.

**Comment by gabriele2000** [comment]
> An option to create a symlink of a file using the right-click context menu.

Agreed, I'd like to paste a shortcut to a file that a lot of programs has to use and I'm not a fan of duplication.
